modules/ROOT/pages/osv2-faq.adoc

```diff
@@ -49,7 +49,17 @@ The total size of an object store is not limited, but each value is limited to 1
 
 == How long can data persist in Object Store v2?
 
-Object Store v2 data persists 30 days. The expiry time is reset if an update to that key occurs. If there is no update, the key is deleted at the 30 day mark.
+The maximum TTL (time to live) is 2,592,000 seconds (30 days).
```

Comment:
Why is TTL spelled here and not in others? Does this appear "first" (although that can be hard to determine)?

Reply:
first instance in this topic.